tools/site_admin/manage_site_access_privileges.php

```diff
@@ -105,20 +106,29 @@ function show_toggles_form($username, $user_settings)
     echo "<form method='POST'>\n";
     echo "<input type='hidden' name='username' value='$username'>\n";
     echo "<input type='hidden' name='action' value='update'>\n";
-    echo "<table>\n";
+    echo "<table class='basic'>\n";
     {
         echo "<tr>\n";
         echo "<th>" . _("Enable") . "</th>\n";
+        echo "<th>" . _("Changed") . "</th>\n";
         echo "<th>" . _("Setting") . "</th>\n";
         echo "<th>" . _("Description") . "</th>\n";
         echo "</tr>\n";
     }
     foreach ( $boolean_user_settings as $setting_name => $setting_description )
     {
         $user_current_value = $user_settings->get_boolean($setting_name);
-        $checked_attr = ($user_current_value == 'yes' ? 'checked' : '');
+        $checked_attr = ($user_current_value ? 'checked' : '');
+
+        $access_log_entry = get_latest_access_change_entry($username, $setting_name);
+        if($access_log_entry)
+            $changed = strftime('%Y-%m-%d %H:%M', $access_log_entry["timestamp"]);
+        else
+            $changed = "<i>" . _("unknown") . "</i>";
```

Comment:
Looking at:
https://www.pgdp.org/~cpeel/c/tools/site_admin/manage_site_access_privileges.php?username=wfarrell 
I'm not sure I like that column of italic "unknown". Could we consider leaving it blank unless an access log entry was found?

Reply:
Sure. I used _unknown_ to be explicit that we really don't know when that was set rather than leave it blank and make the user guess, but I leave that decision to you, @srjfoo, and @lhamilton1.

Reply:
I would prefer to have something there, rather than having it be blank, and <i>unknown</i> makes as much sense as anything to me. It differentiates it from the section of the script that doesn't have a date component associated with it.

One other note -- since enabling a sitemanager is not included in the script (for valid reasons), the sawiki instructions should be updated once the code goes live.